### DISCUSSION TITLE
Save collection entries to FaunaDB etc

### DISCUSSION THREAD

**Discussion by madsem** [decaporg/decap-cms#6234] [Q&A]
Now that Netlify supports database backends, 
what would be the best way to save certain collection entities in for example FaunaDB?

Is the way to go using the cms events?

**Comment by talves** [comment]
I don't feel the fact Netlify supports backends through API's changes the way this would work. It just gives you an option to be able to drive content through another endpoint. Either way, the best option is to create a custom backend plugin that would connect to the service solutions you want to use.

Creating a new backend you can register with Netlify CMS would still require authorization and those endpoints would require a way to decipher where the collection data will end up. This is the beauty of having backends separate from the UI.

If you wanted your backend to be targeted to FaunaDB, I can see a solution using Netlify Graph as a good option when creating the api endpoints, since it would allow for the use of Netlify Identity and you could check auth at that time rather than have to roll your own auth provider inside your customer plugin.

I've written multiple custom backends for Netlify CMS, and in all the time doing it the auth providers ended up being the most time consuming part of the task. I've mostly stuck to serverless solutions like firebase/firestore which already has an auth provider solution, so there's no need for the extra layer of API in that instance.

**TL;DR**: In the end, there's no difference hitting a Netlify API endpoint vs another services endpoint. The custom backend plugin will still need to be created either way.
